### PR TITLE
Fix/4392 remove data hash

### PR DESCRIPTION
**Pull request by AndrewJGaut**
The profiler found that computing data hashes sometimes resulted in very noticeable slowdowns for users. However, we were not using the data hash for anything. This PR removes the data hash computation from the update_user_disk_metadata function and removes all references to data hashes. Also migrates the database to remove the data_hash from column from the Bundle table.

### CODE REVIEW

File: codalab/model/bundle_model.py [hunk 1]
```diff
@@ -1141,7 +1141,6 @@ def update_disk_metadata(self, bundle, bundle_location, enforce_disk_quota=False
             dirs_and_files = [], [bundle_location]
 
         # TODO(Ashwin): make this non-fs specific
```
Comment (epicfaace) [inline codalab/model/bundle_model.py:1143]: rm this line, as path_util.get_size is non-fs specific

File: codalab/model/bundle_model.py [hunk 2]
```diff
@@ -2769,15 +2762,6 @@ def update_user_last_login(self, user_id):
         """
         self.update_user_info({'user_id': user_id, 'last_login': datetime.datetime.utcnow()})
 
-    def _get_disk_used(self, user_id):
```
Comment (epicfaace) [inline codalab/model/bundle_model.py:2772]: This function was unused, correct?

File: codalab/rest/bundles.py [hunk 3]
```diff
@@ -1319,11 +1319,8 @@ def delete_bundles(uuids, force, recursive, data_only, dry_run):
 
     # Delete the actual bundle
     if not dry_run:
-        if data_only:
-            # Just remove references to the data hashes
-            local.model.remove_data_hash_references(relevant_uuids)
-        else:
-            # Actually delete the bundle
+        if not data_only:
+            # Delete bundle metadata.
```
Comment (epicfaace) [inline codalab/rest/bundles.py:1323]: change this comment to "delete the bundle's data"

File: tests/cli/test_cli.py [hunk 4]
```diff
@@ -2220,18 +2269,19 @@ def test_write(ctx):
     check_equals(str(['write\tmessage\thello world']), get_info(uuid, 'actions'))
 
 
+"""
+This we'll have ot think about how to migrate...
```
Comment (epicfaace) [inline tests/cli/test_cli.py:2273]: ? remove?

**Review by epicfaace — APPROVED** [review]

**Comment by ghost** [comment]
> Andrew YMCA CROYDON?

File: codalab/model/bundle_model.py (hunk 1 above)
Comment (ghost) [reply codalab/model/bundle_model.py:1143]: []()

**Pull request merged**